spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
```

Comment:
You shouldn't need to include this (I believe it's already included in the `spec_helper.rb` file).